pkg/scheduler/factory.go

```diff
@@ -208,11 +205,6 @@ func (c *Configurator) CreateFromConfig(policy schedulerapi.Policy) (*Scheduler,
 	if policy.HardPodAffinitySymmetricWeight != 0 {
 		c.hardPodAffinitySymmetricWeight = policy.HardPodAffinitySymmetricWeight
 	}
-	// When AlwaysCheckAllPredicates is set to true, scheduler checks all the configured
-	// predicates even after one or more of them fails.
-	if policy.AlwaysCheckAllPredicates {
-		c.alwaysCheckAllPredicates = policy.AlwaysCheckAllPredicates
```

Comment:
is this removing the code that honors the field? that's not how deprecation periods work... we announce a deprecation, then once the deprecation period or API version is removed, we remove the supporting code.

Reply:
it looks like compatibility was already broken in https://github.com/kubernetes/kubernetes/commit/dc3d1bd238ef460339946185713b59675819ab3f#diff-c237cdd9e4cb201118ca380732d7f361L695-L700 when the behavior driven by this field was removed

Reply:
To restore the behavior, we can plump the flag to the framework and have it examined when ```RunFilterPlugins``` since predicates are now ```Filter``` plugins.

Reply:
PR that restores the behaviour: https://github.com/kubernetes/kubernetes/pull/86496